airbyte-commons-protocol/src/test/java/io/airbyte/commons/protocol/AirbyteMessageSerDeProviderMicronautTest.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.commons.protocol;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import javax.inject.Inject;
+import org.junit.jupiter.api.Test;
+
+@MicronautTest
+class AirbyteMessageSerDeProviderMicronautTest {
+
+  @Inject
+  AirbyteMessageSerDeProvider serDeProvider;
+
+  @Test
+  void testSerDeInjection() {
+    // This should contain the list of all the supported majors of the airbyte protocol
+    final Set<String> expectedVersions = new HashSet<>(List.of("0"));
+
+    assertEquals(expectedVersions, serDeProvider.getDeserializerKeys());
```

Comment:
Nit: probably don't apply to this test because the list are small but for testing sequence I like to use AssertJ.
It provides an output which is easier to read if the assertion fails. Like this is present in list A and not B...

Feel free to use it or not.

Reply:
I personally lean pretty strongly the opposite way, I think the assertThat notation in java is much heavier to read.
This being said, I am fine either way if it helps consistency.

Reply:
I was speaking about the output message for debugging. I don't want to enforce it.